src/microbe_stage/systems/MicrobeVisualsSystem.cs

```diff
@@ -300,7 +300,9 @@ private void CreateOrganelleVisuals(Node3D parentNode, ref OrganelleContainer or
     {
         organelleContainer.CreatedOrganelleVisuals ??= new Dictionary<PlacedOrganelle, Node3D>();
 
-        var organelleColour = PlacedOrganelle.CalculateHSVForOrganelle(cellProperties.Colour);
+        // To make organelles' color dependent on membrane's color:
+        // var organelleColour = PlacedOrganelle.CalculateHSVForOrganelle(cellProperties.Colour);
+        var organelleColour = Colors.White;
```

Comment:
I guess we can see if people prefer organelles to be white tinted always now...

Though I have one question: would something like using like 0.1 multiplier on the HSV colour to blend from white look good? That way we'd still have a bit of species customization on the organelles. I think that would be a less drastic change and maybe would make both groups happy?